templates/launch.html

```diff
@@ -460,6 +463,58 @@ <h3>Runner Data</h3>
         document.getElementById('schema-url-survey-type').selectedIndex = -1;
     }
 
+    function retrieveSchemaName() {
+        const availableSchemas = [...document.querySelector("#schema_name").options].map(x => x.value)
+        let schemaName = localStorage.getItem("schema_name");
+        if (schemaName && availableSchemas.includes(schemaName)) {
+            document.querySelector("#schema_name").value = schemaName;
+
+            toggleSubmitFlushButtons("show");
+            clearAndDisableSchemaUrlSelection()
+
+            const surveyType = document.querySelector(`#schema_name option[value="${schemaName}"]`).dataset.surveyType
+
+            loadSurveyMetadata(schemaName, surveyType)
+            loadSchemaMetadata(schemaName, null)
+            document.querySelector("#local-storage-btn").classList.remove("btn--hidden");
+        }
+        else {
+            document.querySelector("#local-storage-btn").classList.add("btn--hidden");
+        }
+    }
+
+    function retrieveResponseId() {
+        let responseId = localStorage.getItem("response_id");
+
+        if (responseId) {
+            document.querySelector("#response-id-btn").classList.remove("btn--hidden");
+        }
+        else {
+            document.querySelector("#response-id-btn").classList.add("btn--hidden");
+        }
+    }
+
+    function loadResponseId() {
+        document.querySelector("#response_id").value = localStorage.getItem("response_id");
+    }
+
+    function saveResponseId() {
+        localStorage.setItem("response_id", document.querySelector("#response_id").value)
+    }
+
+    function clearLocalStorage() {
+        localStorage.removeItem("response_id");
+        localStorage.removeItem("schema_name");
```

Comment:
We could just refresh the page after this. Then we don't have to keep adding to this.
Button could just be `Clear Local Storage and Refresh`